sys/androidmedia/gstamc.h

```diff
@@ -26,14 +26,14 @@
 #include <gst/audio/multichannel.h>
 #include <gst/androidjni/gstjnimediaformat.h>
 #include <jni.h>
-#include <fluc/drm/flucdrm.h>
+#include "gstamcdrm.h"
 
 G_BEGIN_DECLS
+
 typedef struct _GstAmcCodecInfo GstAmcCodecInfo;
 typedef struct _GstAmcCodecType GstAmcCodecType;
 typedef struct _GstAmcCodec GstAmcCodec;
 typedef struct _GstAmcBufferInfo GstAmcBufferInfo;
-typedef struct _GstAmcCrypto GstAmcCrypto;
```

Comment:
Is the definition of GstAmcCrypto come from somewhere else. I can see it removed from the commit but there is not replacement. It is used all over this files, guessing it must be included from somewhere else or there is a missing file?

Reply:
the replacement is in gstamcdrm.h and gstamcdrm.c
https://github.com/fluendo/gst-plugins-bad/pull/78/files#diff-b32e2c53b0cccac5ea559ac8237a26b6R54
https://github.com/fluendo/gst-plugins-bad/pull/78/files#diff-2671a63b6006077f5804af5f21185394R56